production/scrapers/tennessee_deaths.R

```diff
@@ -0,0 +1,116 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+tennessee_deaths_check_date <- function(url, date = Sys.Date()){
+    z <- get_src_by_attr(
```

Comment:
Let's have `z` be a more descriptive variable name. I think repeating `tn_deaths_pdf` like you've used before is clear.